java/cuvs-java/pom.xml

```diff
@@ -155,29 +160,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-assembly-plugin</artifactId>
-                <version>3.4.2</version>
-                <configuration>
-                    <descriptorRefs>
-                        <descriptorRef>jar-with-dependencies</descriptorRef>
-                    </descriptorRefs>
-                    <mergeManifestMode>merge</mergeManifestMode>
-                    <archiverConfig>
-                        <duplicateBehavior>add</duplicateBehavior>
-                    </archiverConfig>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>assemble-all</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>single</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
This is no longer required. 

The fat-jar is built using `maven-resources-plugin`.